### DISCUSSION TITLE
/v2/zh/guide/feature/search.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#182] [Announcements]
# /v2/zh/guide/feature/search.html

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.gitee.io/v2/zh/guide/feature/search.html

**Comment by zjb-it** [comment]
插件@vuepress/search 不兼容

runtime-core.esm-bundler.js:38 [Vue warn]: injection "Symbol(routeLocale)" not found. 
  at <SearchBox locales= Object hotKeys= Array(2) maxSuggestions=5 > 
  at <SearchBox> 
  at <NavActions showScreen=false onToggleScreen=fn<onToggleScreen> > 
  at <NavBar onToggleSidebar=fn<onToggleSidebar> > 
  at <GlobalEncrypt> 
  at <CommonWrapper > 
  at <Layout> 
  at <AsyncComponentWrapper> 
  at <Vuepress onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< Proxy > > 
  at <RouterView> 
  at <VuepressApp>

**Reply by Mister-Hope** [reply]
自行检查你的依赖树
